src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -24,23 +24,30 @@ const BLOODSPILLER_BLOOD_COST = 50
 // Meters
 // ------
 const MAX_MANA = 9480
+const MANA_REGEN_PERCENT_PER_TICK = 0.02
+const TICK_RATE = 3000
+
 const MAX_BLOOD = 100
 //const MANA_PER_OUT_OF_COMBAT_TICK = 568 // DA should be used at least 2 ticks pre pull
 //const DARKSIDE_MANA_COST = 600
 
-const BLOOD_PRICE_BLOOD_DAMAGE_TRIGGERED_BLOOD_AMOUNT = 1
-const BLOOD_PRICE_BLOOD_DAMAGE_TRIGGERED_MANA_AMOUNT = 120
+const BLOOD_PRICE_DAMAGE_TAKEN_BLOOD_AMOUNT = 1
+const BLOOD_PRICE_DAMAGE_TAKEN_MANA_AMOUNT = 120
 const BLOOD_PRICE_BLOOD_PASSIVE_RATE = 3000
 const BLOOD_PRICE_BLOOD_PASSIVE_AMOUNT = 4
 const BLOOD_PRICE_MAX_DURATION = 15000
 
+const CARVE_AND_SPIT_NO_DA_MANA_GAIN = 1200
```

Comment:
Why not use the `MANA_MODIFIERS` array? Can add `[ACTIONS.CARVE_AND_SPIT.id]: {value: 1200},` for later referencing, since other generators are there alongside spenders.

Reply:
One part me doing a minimal change with the refactor tool in my IDE, and one part C&S doesn't always give the 1200 mana gain, so it didn't "fit" within the MANA_MODIFIERS object the way I was looking at it.  I'm ok with changing to using the MODIFIERS object if that's the preferred approach.